README.md

```diff
@@ -45,31 +45,31 @@ Tart Executor is open sourced under MIT license so people can base their own exe
 Tart itself on the other hand is [source available under Fair Software License](https://tart.run/licensing/)
 that required paid sponsorship upon exceeding a free limit.
 
-## Advanced configuration
+## Supported environment variables
 
-By default, for preparing a virtual machine and running scripts inside of it Tart Executor assumes that
-your virtual machines can be SSH-ed into `admin` user via `admin` password.
-
-```toml
-[runners.custom]
-  prepare_exec = "gitlab-tart-executor"
-  prepare_args = ["prepare", "--username=gitlab", "--password=password101"]
-  run_exec = "gitlab-tart-executor"
-  run_args = ["run", "--username=gitlab", "--password=password101"]
-  cleanup_exec = "gitlab-tart-executor"
-  cleanup_args = ["cleanup"]
-```
-
-`prepare` command has arguments to override CPU/Memory of cloned virtual machines.
-For a full list of options please refer to `gitlab-tart-executor prepare --help`.
+| Name                      | Default | Description                                                                                                                          |
+|---------------------------|---------|--------------------------------------------------------------------------------------------------------------------------------------|
+| `CIRRUS_GTE_SSH_USERNAME` | admin   | SSH username to use when connecting to the VM                                                                                        |
```

Comment:
What do you think of `TART_EXECUTOR` prefix instead of `CIRRUS_GTE`? There is no Cirrus related stuff except the company name. Maybe event just `TART`?

Reply:
Renamed in https://github.com/cirruslabs/gitlab-tart-executor/pull/11/commits/09b188356e7c24e748eeac8c28ad1a3255ad2a50 👌